bugshaker/src/main/java/com/github/stkent/bugshaker/MainActivity.java

```diff
@@ -0,0 +1,279 @@
+	public void onClick(View view){
+		//respond to clicks
+		if(view.getId()==R.id.draw_btn){
+			//draw button clicked
+			final Dialog brushDialog = new Dialog(this);
+			brushDialog.setTitle(getApplicationContext().getString(R.string.brush_dialog_title));
+			brushDialog.setContentView(R.layout.brush_chooser);
+
+			ImageButton smallBtn = (ImageButton) brushDialog.findViewById(R.id.small_brush);
+			smallBtn.setOnClickListener(new OnClickListener() {
+				@Override
+				public void onClick(View view) {
+					drawView.setBrushSize(smallBrush);
+					drawView.setLastBrushSize(smallBrush);
+					drawView.setErase(false);
+					brushDialog.dismiss();
+				}
+			});
+
+			ImageButton mediumBtn = (ImageButton) brushDialog.findViewById(R.id.medium_brush);
+			mediumBtn.setOnClickListener(new OnClickListener() {
+				@Override
+				public void onClick(View view) {
+					drawView.setBrushSize(mediumBrush);
+					drawView.setLastBrushSize(mediumBrush);
+					drawView.setErase(false);
+					brushDialog.dismiss();
+				}
+			});
+
+			ImageButton largeBtn = (ImageButton) brushDialog.findViewById(R.id.large_brush);
+			largeBtn.setOnClickListener(new OnClickListener() {
+				@Override
+				public void onClick(View view) {
+					drawView.setBrushSize(largeBrush);
+					drawView.setLastBrushSize(largeBrush);
+					drawView.setErase(false);
+					brushDialog.dismiss();
+				}
+			});
+
+			brushDialog.show();
+
+		} else if (view.getId() == R.id.erase_btn){
+			//switch to erase, choose size
+			final Dialog brushDialog = new Dialog(this);
+			brushDialog.setTitle(getApplicationContext().getString(R.string.brush_dialog_title));
+			brushDialog.setContentView(R.layout.brush_chooser);
+			ImageButton smallBtn = (ImageButton) brushDialog.findViewById(R.id.small_brush);
+			smallBtn.setOnClickListener(new OnClickListener() {
+				@Override
+				public void onClick(View view) {
+					drawView.setErase(true);
+					drawView.setBrushSize(smallEraser);
+					brushDialog.dismiss();
+				}
+			});
+
+			ImageButton mediumBtn = (ImageButton) brushDialog.findViewById(R.id.medium_brush);
+			mediumBtn.setOnClickListener(new OnClickListener() {
+				@Override
+				public void onClick(View view) {
+					drawView.setErase(true);
+					drawView.setBrushSize(mediumEraser);
+					brushDialog.dismiss();
+				}
+			});
+
+			ImageButton largeBtn = (ImageButton) brushDialog.findViewById(R.id.large_brush);
+			largeBtn.setOnClickListener(new OnClickListener() {
+				@Override
+				public void onClick(View view) {
+					drawView.setErase(true);
+					drawView.setBrushSize(largeEraser);
+					brushDialog.dismiss();
+				}
+			});
+			brushDialog.show();
+		}
+
+		else if (view.getId()==R.id.sendEmail){
+			saveDrawing();
+		}
+		else if (view.getId()==R.id.speechBox){
+
+			createTextEdit();
+		}
+
+
+
+	}
+
+	private void createTextEdit(){
+		//get x and y
+
+	}
+
+	public void paintClicked(View view){
+		//use chosen color
+		drawView.setBrushSize(drawView.getLastBrushSize());
```

Comment:
This one doesn't set the color. Also, we usually avoid putting any comments in code. Our general practice is to name methods (and variables) and write code in such a way that is self explanatory!